lib/kino/js/live.ex

```diff
@@ -202,44 +202,8 @@ defmodule Kino.JS.Live do
 
   See `c:GenServer.init/1` for more details.
 
-  > #### Starting other kinos {: .warning}
-  >
-  > It is generally not possible to start kinos inside the `c:init/2`
-  > callback, as such operation would block forever. In case you need
-  > to start other kinos during initialization, you must start them
-  > beforehand and pass as an argument to `c:init/2`. So instead of
-  >
-  >     defmodule KinoDocs.Custom do
-  >       def new() do
-  >         Kino.JS.Live.new(__MODULE__, {})
-  >       end
-  >
-  >       @impl true
-  >       def init({}, ctx) do
-  >         frame = Kino.Frame.new()
-  >         {:ok, assign(ctx, frame: frame)}
```

Comment:
This example is confusing, we create a frame in `Kino.JS.Live`, but there is no obvious way how we would use that frame (since `Kino.JS.Live` itself is already about managing JS rendering). If we want a frame to render updates into, as we do in the dbg pipeline, it is actually more natural to create the frame upfront and pass as an argument.

I think it's more about creating kinos like `Kino.DataTable.new()` and rendering into a provided frame, and it's also more about `GenServer`, because it's a rare case for `Kino.JS.Live`.